packages/clarity-js/src/data/metadata.ts

```diff
@@ -41,11 +41,13 @@ export function start(): void {
 
   if (navigator) {
     dimension.log(Dimension.Language, (<any>navigator).userLanguage || navigator.language);
+    metric.max(Metric.Automation, navigator.webdriver ? BooleanFlag.True : BooleanFlag.False);
```

Comment:
There's a bunch of stuff for detection automation/headless browser, etc. Is that the goal of this or is it more restricted in inattention?